pkg/operation/care/seed_health.go

```diff
@@ -81,6 +94,9 @@ func (h *SeedHealth) checkSeedSystemComponents(
 	if gardencorev1beta1helper.SeedSettingDependencyWatchdogProbeEnabled(h.seed.Spec.Settings) {
 		managedResources = append(managedResources, dependencywatchdog.ManagedResourceDependencyWatchdogProbe)
 	}
+	if gardencorev1beta1helper.SeedUsesNginxIngressController(h.seed) {
```

Comment:
We need a similar check for VPA, because it can be enabled/disabled in the seed spec, right?

Reply:
VPA is a bit different, the `vpa-exporter` part still gets deployed even if it is disabled, so the managed resource will still be present in the seed: 
https://github.com/gardener/gardener/blob/e2b5d7c4c3ec44b9ecb674dd8c898a4bbcc68905/pkg/operation/botanist/component/vpa/vpa.go#L97-L99
https://github.com/gardener/gardener/blob/e2b5d7c4c3ec44b9ecb674dd8c898a4bbcc68905/pkg/operation/botanist/component/vpa/vpa.go#L143-L153

Reply:
Ah perfect, thanks!